### ISSUE TITLE
Ceph versioning graphic is inaccurate

### ISSUE THREAD

**Issue by pmatulis** [canonical-web-and-design/ubuntu.com#7979]
The below graphic on https://ubuntu.com/ceph is inaccurate:

![image](https://user-images.githubusercontent.com/6303808/88194403-c8dfb780-cc0c-11ea-9f14-e427152c370e.png)

It shows OpenStack Stein as being associated with Ceph Nautilus. This is wrong. Stein provides [13.2.9](https://docs.ceph.com/docs/master/releases/mimic/#v13-2-9-mimic), which is Mimic.

```
cmadison ceph -u cloud-archive G stein
 ceph | 13.2.9-0ubuntu0.19.04.1~cloud0  | stein           | bionic-updates  | source
 ceph | 13.2.9-0ubuntu0.19.04.1~cloud0  | stein-proposed  | bionic-proposed | source
```

---
*Reported from: https://ubuntu.com/ceph*

**Comment by pmahnke** [comment]
![image](https://user-images.githubusercontent.com/441217/88266562-4c5ddf00-ccc7-11ea-9ab3-d1b418cdaa4d.png)

**Comment by pmahnke** [comment]
@waynecrosby do you have the original artwork?  could you provide a new version?

**Comment by waynecrosby** [comment]
@pmahnke Yes we have the artwork so I can amend for you. Just to clarify, I will just reduce the size of Ceph Nautilus and extend Ceph Mimic - - I don't extend Ceph Mimic and shift Ceph Nautilus & Ceph Octopus down to the end?

**Comment by pmahnke** [comment]
@waynecrosby that's correct

**Comment by waynecrosby** [comment]
@pmahnke Here is the updated visual and link from the asset server:
<img width="798" alt="Benefits-of-ceph-on-ubuntu-2020-opt1" src="https://user-images.githubusercontent.com/42238722/88268434-5cc38900-ccca-11ea-880a-7e40d1594c28.png">

Asset link:
https://assets.ubuntu.com/v1/e76a9b6d-Benefits-of-ceph-on-ubuntu-2020-opt1.svg

**Comment by pmatulis** [comment]
Thanks for the quick turnaround on this.
